### PR TITLE
Multiple fixes for report templates

### PR DESCRIPTION
**Pull request by aj-he**
addresses the following report template issues

#796
#799
#804
#806
#808
#813
#816
#817
#825
#826
#827
#830
#831
#832
#833
#834
#842
#846
#860
#863
#864
#878
#885
#922 (Merge remote-tracking branch 'origin/42415_ro_use_phases_start-date-displaying-end-date-value' into report_template_fixes)
#923
#915 (fix for related resources)

### CODE REVIEW

**Comment by aj-he** [comment]
Hi @chiatt, apologies for the size of this PR. They are only small changes but go across a lot of issues so I thought that as this content was large it would be best for you to review, so you guys understand what is going in. The others are smaller so should be easier for you to track from afar.

File: arches_her/media/js/views/components/reports/scenes/resources.js [hunk 1]
```diff
@@ -57,94 +64,108 @@ define(['underscore', 'knockout', 'arches', 'utils/report','bindings/datatable']
...
-                const associatedArtifactsNode = self.getRawNodeValue(params.data(), self.dataConfig.assets);
+                const associatedArtifactsNode = self.getRawNodeValue(params.data(), self.dataConfig.assets, 'instance_details');
                 if(Array.isArray(associatedArtifactsNode)){
                     self.assets(associatedArtifactsNode.map(x => {
-                        const resourceName = self.getNodeValue(x, {
-                            testPaths: [['associated heritage asset, area or artefact'],['heritage asset, area or artefact'], []]
-                         });
-                        const association = self.getNodeValue(x, 'association type'); 
+                        var resource = [];
+                        for (const element of x?.['Associated Heritage Asset, Area or Artefact']?.['instance_details']) {
```
Comment (chiatt) [inline arches_her/media/js/views/components/reports/scenes/resources.js]: While testing #863, I ran into this error at line 122: 
image

**Review by chiatt — COMMENTED** [review]
Overall this looks great. I just saw one error which I've commented on inline. Also, I noticed that #847 is listed in the addressed tickets, but it doesn't look like it's resolved.

**Review by njkim — CHANGES_REQUESTED** [review]
#825 "Associated Heritage Asset, Area or Artefact" still not showing, (associate file seems to be fixed.
#827 Activity model location tab is still not working in report view, while it is working in the editor view. (Other models' locations seem to be working properly)

**Comment by aj-he** [comment]
@chiatt @njkim thanks for reviewing. There is another ticket #915 that @RobOatesHistoricEngland is working on which is a broad fix for the Associated Resources tabs across multiple models that I think will fix these remaining issues. I'll be reviewing it with him in the morning and will then add it to this PR. There has been a general issue with cardinality being different for data going into shared scenes, so hopefully this will fix it.

**Comment by aj-he** [comment]
> Overall this looks great. I just saw one error which I've commented on inline. Also, I noticed that #847 is listed in the addressed tickets, but it doesn't look like it's resolved.

@chiatt you are right, I think this is a new feature rather than a bug. I have removed #847 from the resolved tickets list.

File: arches_her/media/js/views/components/reports/scenes/resources.js (hunk 1 above)
Comment (aj-he) [reply arches_her/media/js/views/components/reports/scenes/resources.js]: This is now resolved from #915

**Comment by aj-he** [comment]
> #825 "Associated Heritage Asset, Area or Artefact" still not showing, (associate file seems to be fixed. #827 Activity model location tab is still not working in report view, while it is working in the editor view. (Other models' locations seem to be working properly)

@njkim these are no longer an issue from my testing of fixes for #915. Please can you check your end?

**Review by chiatt — APPROVED** [review]
Looks good to me!

**Review by njkim — APPROVED** [review]
Look great

**Pull request merged**